runtime/j9vm/java11vmi.c

```diff
@@ -663,6 +669,7 @@ allowReadAccessToModule(J9VMThread * currentThread, J9Module * fromModule, J9Mod
 					fromModule->removeAccessHashTable = vm->internalVMFunctions->hashModuleTableNew(vm, INITIAL_INTERNAL_MODULE_HASHTABLE_SIZE);
 				}
 				if (NULL != fromModule->removeAccessHashTable) {
+					Trc_MODULE_invokeHashTableAtPut(currentThread, "allowReadAccessToModule(removeAccessHashTable)", fromModule->removeAccessHashTable, &toModule, toModule, "false");
```

Comment:
fromModule here?

Reply:
Fixed.

@DanHeidinga the PR has been updated accordingly, please have another look.